CentralDeErros.Data/CentralDeErrosDbContext.cs

```diff
@@ -0,0 +1,38 @@
+﻿using CentralDeErros.API.Models;
+using CentralDeErros.Core.Models;
+using CentralDeErros.Core.Models.Maps;
+using Microsoft.EntityFrameworkCore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CentralDeErros.Data
+{
+    public class CentralDeErrosDbContext : DbContext
+    {
+        public DbSet<Microsservice> Microsservices { get; set; }
+        public DbSet<Error> Errors { get; set; }
+        public DbSet<Occurrence> Occurrences { get; set; }
+        public DbSet<Level> Levels { get; set; }
+        public DbSet<User> Users { get; set; }
+        public DbSet<Perfil> Perfils { get; set; }
+
```

Comment:
Faltou incluir o Dbset da model Envorinment, 

Mas qq coisa incluo quando for fazer manhã.

Reply:
Show! Posso fazer aqui tb!